cli/replicaset/roles.go

```diff
@@ -5,18 +5,57 @@ import (
 	"slices"
 )
 
-// ChangeRoleFunc is a function type for addition or removing a role.
+// RoleAction is a type that describes an action that will
+// happen with Roles. It can be "add" or "remove".
+type RoleAction uint
+
+const (
+	AddAction RoleAction = iota
+	RemoveAction
+)
+
+// ChangeRoleFunc is a function for adding/removing roles.
```

Comment:
Why do we need a function if we have an interface?

Reply:
To name interface's method in function argument to not pass a whole set of methods. For example: 

https://github.com/tarantool/tt/blob/29379627ec9c41c63ce3d99e6791e6bd23ba20a8/cli/replicaset/cconfig.go#L755-L756

Reply:
Interfaces should not contain many methods, so I don't see any problem. Adding a function type for this is unnecessary.

Reply:
Removed function declaration and change arg on interface in related functions.